CMakeLists.txt

```diff
@@ -393,7 +393,12 @@ ELSE(CMAKE_TOOLCHAIN_FILE)
     MESSAGE(WARNING "No compiler identified")
   ENDIF()
   MESSAGE(STATUS "Compiler: ${COMPILER}")
-  MESSAGE(STATUS "Compiler Version: ${CMAKE_CXX_COMPILER_VERSION}")
+  # Workaround for not properly identified CXX compiler in case of IBM XL compiler
+  IF( ${COMPILER} MATCHES "IBM" )
+    MESSAGE(STATUS "Compiler Version: ${CMAKE_C_COMPILER_VERSION}")
+  ELSE()
+    MESSAGE(STATUS "Compiler Version: ${CMAKE_CXX_COMPILER_VERSION}")
+  ENDIF()
 
```

Comment:
I was wrong. DO NOT move it in CMake/IBMCompilers.cmake.
Somehow we already have printout in the very beginning, for example.
 -- The C compiler identification is GNU 6.2.0
-- The CXX compiler identification is GNU 6.2.0
Here we print again.
-- Compiler: GNU
-- Compiler Version: 6.2.0
Unless they can be different, we can just remove the printout here.